tests/collections/collection.test.ts

```diff
@@ -16,14 +16,23 @@ import assert from 'assert';
 import { Db } from '@/src/collections/db';
 import { Collection } from '@/src/collections/collection';
 import { Client } from '@/src/collections/client';
-import { testClient, testClientName, sampleUsersList, createSampleDocWithMultiLevel, createSampleDocWithMultiLevelWithId, TEST_COLLECTION_NAME } from '@/tests/fixtures';
+import {
+    testClient,
+    testClientName,
+    sampleUsersList,
+    createSampleDocWithMultiLevel,
+    createSampleDoc2WithMultiLevel,
+    createSampleDoc3WithMultiLevel,
+    createSampleDocWithMultiLevelWithId,
+    TEST_COLLECTION_NAME
+} from '@/tests/fixtures';
 
 describe(`StargateMongoose - ${testClientName} Connection - collections.collection`, async () => {
     const isAstra: boolean = testClientName === 'astra';
     let astraClient: Client | null;
     let db: Db;
     let collection: Collection;
-    before(async function () {
+    before(async function() {
```

Comment:
I'd recommend being careful about using your own code formatting setup in your IDE, because it may not line up with what the stargate-mongoose repo uses. And that will lead to hard-to-review PRs.

Reply:
sorry, about that. 
` npx eslint --fix tests/collections/collection.test.ts `
I used this one to format the file.
Please advise what should I use, Thanks

Reply:
That command should work fine, I'm not sure why your ESLint made all these changes. On my machine, running `npx eslint --fix tests/collections/collection.test.ts` doesn't make any changes.

In general, I'd recommend just running `npm run lint -- --fix` instead of npx, just to make sure you're using the expected version of ESLint.